test-data/unit/fixtures/typing-typeddict.pyi

```diff
@@ -43,7 +43,7 @@ class Iterator(Iterable[T_co], Protocol):
     def __next__(self) -> T_co: pass
 
 class Sequence(Iterable[T_co]):
-    def __getitem__(self, n: Any) -> T_co: pass
+    def __getitem__(self, n: Any) -> T_co: pass # type: ignore
```

Comment:
Why is this `# type ignore`d? Perhaps leave a note for future viewers?

Reply:
It has an Any and my new test case checks for explicit Any with typeddict stubs. Alternatively I could add silent import flag to the test case.

Reply:
What you now makes sense then, I suppose putting a specific code on the `type: ignore` (like, if you run with `--show-error-codes` it will tell you what code to put) would be appreciated. But I don't know if that's the style tests normally take so...

Reply:
> I suppose putting a specific code on the type: ignore [...] would be appreciated

+1

Reply:
I've added the error code. The code is vague [misc] so I included a comment too on which error it was.